lib/explorer/series.ex

```diff
@@ -1369,6 +1369,48 @@ defmodule Explorer.Series do
   @spec at(series :: Series.t(), idx :: integer()) :: any()
   def at(series, idx), do: fetch!(series, idx)
 
+  @doc """
+  Returns a string series with all values ​​concatenated.
+
+  ## Examples
+
+      iex> s1 = Explorer.Series.from_list(["a", "b", "c"])
+      iex> s2 = Explorer.Series.from_list(["d", "e", "f"])
+      iex> s3 = Explorer.Series.from_list(["g", "h", "i"])
+      iex> Explorer.Series.format([s1, s2, s3])
+      #Explorer.Series<
+        Polars[3]
+        string ["adg", "beh", "cfi"]
+      >
+
+      iex> s1 = Explorer.Series.from_list(["a", "b", "c", "d"])
+      iex> s2 = Explorer.Series.from_list([1, 2, 3, 4])
+      iex> s3 = Explorer.Series.from_list([1.5, :nan, :infinity, :neg_infinity])
+      iex> Explorer.Series.format([s1, "/", s2, "/", s3])
+      #Explorer.Series<
+        Polars[4]
+        string ["a/1/1.5", "b/2/NaN", "c/3/inf", "d/4/-inf"]
+      >
+
+      iex> s1 = Explorer.Series.from_list([<<1>>, <<239, 191, 19>>], dtype: :binary)
+      iex> s2 = Explorer.Series.from_list([<<3>>, <<4>>], dtype: :binary)
+      iex> Explorer.Series.format([s1, s2])
+      ** (RuntimeError) External error: invalid utf-8 sequence
+  """
+  @doc type: :shape
+  @spec format([
+          Series.t() | number() | Date.t() | Time.t() | NaiveDateTime.t() | boolean() | String.t()
+        ]) :: Series.t()
+  def format([_ | _] = list), do: Shared.apply_series_impl(:format, [cast_to_string(list)])
+
+  defp cast_to_string(list) do
+    Enum.map(list, fn
+      %Series{dtype: :string} = s -> s
+      %Series{} = s -> cast(s, :string)
+      value -> from_list([value], dtype: :string)
```

Comment:
```suggestion
      value when is_binary(value) -> from_list([value], dtype: :string)
      other -> raise ArgumentError, "format/1 expects a list of series or strings, got: #{inspect(other)}"
```

Reply:
I don't think we need to allow all data types. Only series and strings, that should simplify/remove some of the tests too!

Reply:
https://github.com/elixir-nx/explorer/pull/520/commits/6226df4589db02f568930592b34d12c3424b88a4